docs/pages/guides/customizing-metro.md

```diff
@@ -26,6 +28,104 @@ You can find a complete list of supported options in the Metro docs: [Configurin
 
 To add to a value, such as an array of file extensions, defined in the default configuration, you can access the defaults using the `getDefaultConfig(projectRoot)` function defined in `@expo/metro-config`. Add `@expo/metro-config` to the dependencies of your project to do this. In SDK +41 you can import from `expo/metro-config`.
 
+## Web Support
+
+> Metro web support is an experimental feature in Expo SDK 46 and only works with the [local Expo CLI](https://blog.expo.dev/new-versioned-expo-cli-cf6e10632656).
+
+By default, Expo CLI uses Webpack as the bundler on web platforms, this is because Metro historically did not support web.
+Using different bundlers across platforms leads to some critical divergence in how your app works across platforms. Features like Fast Refresh which work on native don't work on web, and important production functionality like assets are treated differently across bundlers.
```

Comment:
```suggestion
By default, Expo CLI uses Webpack as the bundler on web platforms, this is because Metro historically did not support web. Using different bundlers across platforms leads to some critical divergence in how your app works across platforms. Features like Fast Refresh which work on native don't work on web, and important production functionality like assets are treated differently across bundlers.
```

I think it's the prettier formatting not working here. I think those two sentences mean to be in the same paragraph.